### PR TITLE
add meeting notes from 11 February 2020

### LINKED ISSUE #246: 2020.02.11 Collab Summit Planning Meeting
## Time

**UTC Tue 11-Feb-2020 19:00 (06:00 PM)**:

| Timezone      | Date/Time             |
|---------------|-----------------------|
| US / Pacific  | Tue 11-Feb-2020 10:00 (10:00 AM) |
| US / Mountain | Tue 11-Feb-2020 11:00 (11:00 AM) |
| US / Central  | Tue 11-Feb-2020 12:00 (12:00 PM) |
| US / Eastern  | Tue 11-Feb-2020 13:00 (01:00 PM) |
| London        | Tue 11-Feb-2020 18:00 (06:00 PM) |
| Amsterdam     | Tue 11-Feb-2020 19:00 (07:00 PM) |
| Moscow        | Tue 11-Feb-2020 21:00 (9:00 PM) |
| Chennai       | Wed 11-Feb-2020 23:30 (11:30 PM) |
| Hangzhou      | Wed 12-Feb-2020 02:00 (02:00 AM) |
| Tokyo         | Wed 12-Feb-2020 03:00 (03:00 AM) |
| Sydney        | Wed 12-Feb-2020 05:00 (05:00 AM) |

Or in your local time:
* http://www.wolframalpha.com/input/?i=06PM+UTC%2C+Feb+11%2C+2020+in+local+time

## Links

* Rolling Notes Doc: <https://docs.google.com/document/d/166t3D87CGtXpMV-ouYP9TBq2x3pZ16QackYtASPIY5g/edit>

## Agenda

1. **Meta Issues**
2. **Venue & Date Updates**
3. **Registration**
4. **Summit Programming**

## Invited

* OpenJS Foundation Cross Project Council
* OpenJS Foundation Project Maintainers & Project Community Members
* OpenJS Foundation Board of Directors

## Joining the meeting

link for participants: Zoom link: https://zoom.us/j/742726725

---

**Invitees**

Please use the following emoji reactions in this post to indicate your
availability.

* :+1: - Attending
* :-1: - Not attending
* :confused: - Not sure yet

### PR DESCRIPTION
**Pull request by jorydotcom**
Closes #246

### CODE REVIEW

File: meetings/2020-02-11.md [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+## 2020-02-11 Summit Planning Meeting
+
+**Present:** Manil, Ruy, Christian, Rachel, Jory, Joe, Wes
+
+**Agenda**
+
+1. **Venue & Date Updates**
+    1. 2 Member spaces reported back - not available. Looking into others.
+    2. LF Events team looking into whether there is any space that can be recaptured from the conference events on thurs-fri
+    3.  Idea from Wes: Capitol Factory.
```
Comment (wesleytodd) [inline meetings/2020-02-11.md]:
```suggestion
    3.  Idea from Wes: Capital Factory. 
```

File: meetings/2020-02-11.md [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+    3. Nominally set price for $15 (exceptions available)
+        * will plan to have registration process, namely to get a firm number of attendees. To be discussed at a future meeting.
+4. **Summit Programming**
+    1. Research Question: Would your project be interested in holding its own collab summit on the same day in the same city as the OpenJSF Collab Summit?
+        12. Answer was generally yes! But we may not have time to pull that off for June 2020. Discussion / Interest in having project 'tracks' seems possible.[https://docs.google.com/spreadsheets/d/1TjTw5An3eTLP735cu7mNEFAoyd19p1ua54HhH57YIcI/edit#gid=0](https://docs.google.com/spreadsheets/d/1TjTw5An3eTLP735cu7mNEFAoyd19p1ua54HhH57YIcI/edit#gid=0)
+    12. Discussion about friendliness/accessibility to new project contributors.
+        * would like to schedule a 'new community member orientation' track - getting started in open source, getting to know the Foundation community, etc.
+        * Could include brief sessions from project maintainers to introduce their project
+        * Encourage maintainers who are speaking to do a 'getting started' session after their talk
+        * emphasize workshops, learning
```
Comment (wesleytodd) [inline meetings/2020-02-11.md]:
```suggestion
        * emphasize workshops, learning
        * Could these run during the main conference?
```

**Comment by ruyadorno** [comment]
👍

File: meetings/2020-02-11.md [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+        * Could these run during the main conference?
+    3. Discussion about meta-topics - organizing session calls around cross-project relevant themes (best practices, security)
+    4. Idea: Put calls out for session topics; attempt to have majority in by late April & plan to fill in from there.
+    5. Idea: Don't require session facilitator/leader in order to encourage people to surface discussions they would like to have
```
Comment (mcollina) [inline meetings/2020-02-11.md]: From previous summits experience, having a facilitator for most session means that the sessions and topics are prepared up-front and curated. I highly discourage removing this.

We might want to keep a few slots as "unconference" / "open" if we want to have more spontaneous discussions

Comment (jorydotcom) [reply meetings/2020-02-11.md]: @mcollina since this is just the meeting notes, I'll pull this out into a separate issue so we can continue the thread :D

Comment (jorydotcom) [reply meetings/2020-02-11.md]: Captured in #250

**Pull request merged**